src/test/functional/features/list-contact-types.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Contact Types List
+
+  Background:
+    Given I am on new browser
+    Given I am on FACT homepage
+    And I am on the admin portal sign in page
+    When I fill in the Username and Password fields with my super user authenticated credentials
+    And click the Sign In button
+    When I click on lists link
+    Then I am redirected to the "Edit A List" page
+    When I hover over the tab title
+    And I click on contact type list
+
+  Scenario: Edit Contact Type
+    Given I click edit contact type "Adoption"
+    Then I am redirected to the contact type "Editing Contact Type: Adoption" form
+    Then I will make sure to clear entries for the Contact Type
+    Then I enter "Adoption" in name textbox
+    Then I enter "Mabwysiadu" in name welsh textbox
+    When I click Contact Type save button
+    Then A green message is displayed for the updated Contact Type "Contact Types Updated"
+
+  Scenario: Add new Contact Type with the name already exist
```

Comment:
If we try and edit an existing contact type, and we change the name to one that already exists, will that also throw an error. Is it worth us checking for that too? 

For example: 

contact types are: 
1, 
2, 
3

We amend contact type 3, and change it to 2, so we have: 

1,
2,
2

That should fail right?

Reply:
Yes, it should fail. I'll add a test to check the scenario. Thanks @Adam